android/app/src/main/java/edu/berkeley/eecs/e_mission/data_sync/SyncingInterface.java

```diff
@@ -0,0 +1,21 @@
+package edu.berkeley.eecs.e_mission.data_sync;
+
+import android.accounts.Account;
+import android.content.ContentProviderClient;
+import android.content.SyncResult;
+import android.os.Bundle;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+
+/**
+ * Created by RZarrabi on 6/15/2015.
+ */
+public interface SyncingInterface {
+    public void onPerformSync(Account account, Bundle extras, String authority,
```

Comment:
Great job creating an interface. But why these two specific methods? If you already have an `onPerformSync`, then why do you need an `updateResultsSummary` as well? I think that in the current code, `updateResultsSummary` is called from `onPerformSync`. I would suggest picking something that makes sense and then changing the existing code to fit it, as opposed to basing the interface on the current code.

So some examples might be:

1. A single `onPerformSync` that does both pull and push 
1. Two separate `pull` and `push` methods so that we can deal separately with the two kinds of failures.